src/lib/viewers/archive/ArchiveExplorer.js

```diff
@@ -85,13 +64,12 @@ class ArchiveExplorer extends React.Component {
      */
     getItemList = (itemCollection, fullPath) => {
         const folderInfo = itemCollection.find(item => item.absolute_path === fullPath);
-        const subItems = getProp(folderInfo, 'item_collection.entries');
+        const subItems = folderInfo.item_collection;
         if (!subItems) {
```

Comment:
It seems like this can be simplified a bit with a default value:

```
const { item_collection: folderItems = [] } = itemCollection.find(item => item.absolute_path === fullPath);
return itemCollection.filter(item => folderItems.includes(item.absolute_path));
```